modules/ocf_kubernetes/files/mastodon-pv.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: mastodon-nfs-pv
+spec:
+  capacity:
+    storage: 30Gi
```

Comment:
This is okay for testing, but once we have a server running and people are uploading pictures and such, we probably want to bump this.

Reply:
30GB seems like a lot. In however long we've been running mysql, it's still < 5 GB. This is probably safe.

Reply:
fair enough